pipelines/DarkEnergyCamera/ApPipeMultiTractFakes.yaml

```diff
@@ -0,0 +1,26 @@
+description: DECam AP Pipeline with synthetic/fake sources. Templates are inputs.
+# Remember:
+# (0) Ensure median calibration products and template coadds exist for the data being processed
+# (1) Run $AP_PIPE_DIR/pipelines/DarkEnergyCamera/RunIsrForCrosstalkSources.yaml
+# (2) Execute `make_apdb.py`, e.g.,
+#     make_apdb.py -c isolation_level=READ_UNCOMMITTED
+#                  -c db_url="sqlite:////project/user/association.db"
+# (3) Run this pipeline, setting appropriate diaPipe configs
+#     (diaPipe configs should match the make_apdb.py configs)
+
+instrument: lsst.obs.decam.DarkEnergyCamera
+imports:
+  - location: $AP_PIPE_DIR/pipelines/ApPipeMultiTractFakes.yaml
+    exclude:  # These tasks frome from DECam's ApPipe.yaml instead
+      - processCcd
+  - location: $AP_PIPE_DIR/pipelines/DarkEnergyCamera/ApPipe.yaml
+    exclude:  # These tasks come from ApPipeMultiTractFakes.yaml instead
+      - imageDifference
+      - diaPipe
+      - transformDiaSrcCat
+
+tasks:
+  diaPipe:
+    class: lsst.ap.association.DiaPipelineTask
+    config:
+      # apdb.db_url: YOU MUST CONFIGURE THIS BEFORE RUNNING THE PIPELINE
```

Comment:
No, it doesn't wipe the `DiaPipelineTask` config - it's there to remind users to configure the db_url at runtime, since there's not any other way to (easily) do this when running via bps.